locales/es-ES/messages.po

```diff
@@ -657,9 +744,21 @@ msgstr ""
 msgid "Import a QTI/APIP XML Item Document"
 msgstr ""
 
+msgid "Import item content and metadata from CSV file. Only choice interactions are supported."
+msgstr ""
+
 msgid "Import QTI ITEM into \"%s"
 msgstr ""
 
+msgid "IMS QTI Item referenced as \"%s\" cannot be imported."
+msgstr ""
+
+msgid "IMS QTI Item referenced as \"%s\" contains a portable element and cannot be imported."
+msgstr ""
+
+msgid "IMS QTI Item referenced as \"%s\" in the IMS Manifest file could not be imported."
+msgstr ""
+
 msgid "In Progress"
 msgstr ""
```

Comment:
`Elección` for `choice interactions` as in https://github.com/oat-sa/extension-tao-itemqti/blob/ec1e8fc53339d25efa911487d5be0297e6c074c8/locales/es-ES/messages.po#L282

```suggestion
msgid "Import item content and metadata from CSV file. Only choice interactions are supported."
msgstr "Importar contenido del item y metadatos de un archivo CSV. Únicamente estás soportadas las interacciones de elección"

msgid "Import QTI ITEM into \"%s"
msgstr "importar Item QTI en \"%s"

msgid "IMS QTI Item referenced as \"%s\" cannot be imported."
msgstr "El item IMS QTI referenciado como \"%s\"  no puede ser importado."

msgid "IMS QTI Item referenced as \"%s\" contains a portable element and cannot be imported."
msgstr "El item IMS QTI referenciado como \"%s\" contiene un elemento portable y no puede ser importado."

msgid "IMS QTI Item referenced as \"%s\" in the IMS Manifest file could not be imported."
msgstr "El item IMS QTI referenciado como \"%s\" en el archivo Manifest IMS no pudo importarse."

msgid "In Progress"
msgstr "En progreso"
```